include/onnxruntime/core/session/onnxruntime_cxx_inline.h

```diff
@@ -287,14 +287,17 @@ inline void IoBinding::ClearBoundOutputs() {
 }
 
 inline Env::Env(OrtLoggingLevel default_warning_level, _In_ const char* logid) {
+  ThrowOnError(GetApi().SetLanguageProjection(p_, OrtLanguageProjection::ORT_PROJECTION_CPLUSPLUS));
```

Comment:
>(p_ [](start = 45, length = 3)

The environment is not created at this point and p_ is not initialized. Same for all other places.